### DISCUSSION TITLE
Unexpected Close Tab Prompt for 6+ Tabs

### DISCUSSION THREAD

**Discussion by agwr** [ghostty-org/ghostty#3256] [Feature Requests, Ideas] [answered]
#### Description
When opening 6 or more horizontal tabs in Ghostty on a Mac with a 3024x1964 display, pressing `cmd+w` prompts the user to confirm closing the tab. For fewer than 6 tabs, `cmd+w` closes the tab without any prompt as expected.

#### Steps to Reproduce
1. Open Ghostty on a Mac with a 3024x1964 display.
2. Open 6 horizontal tabs with `cmd+shift+d`.
3. Press `cmd+w` to close a tab.

#### Expected Behavior
- The tab should close immediately without a prompt, regardless of the number of open tabs.

#### Actual Behavior
- A confirmation prompt appears when there are 6 or more tabs.

#### Environment
- **Ghostty version:** Ghostty 1.0.0
- **OS:** macOS Sonoma 14.5
- **Display Resolution:** 3024x1964

**Comment by mitchellh** [comment]
Not reproducible on my end. Into issue triage discussions until more information can be discovered.

**Comment by 00-kat** [comment]
Possibly related to #3057.

**Comment by mitchellh** [comment, marked as answer]
We haven't heard further reports since Ghostty 1.2. This issue is very old. If this issue is still happening in specific cases, please open a new discussion.
